python/pyspark/ml/feature.py

```diff
@@ -86,7 +86,9 @@ class Binarizer(JavaTransformer, HasInputCol, HasOutputCol, JavaMLReadable, Java
     """
 
     threshold = Param(Params._dummy(), "threshold",
-                      "threshold in binary classification prediction, in range [0, 1]",
+                      "Param for threshold used to binarize continuous features. " + 
+"The features greater than the threshold will be binarized to 1.0. " +  
```

Comment:
All of the lines would need to indent -- these are still flush with the left side.

Reply:
Just indented. Please let me know if I need to fix anything else!